shopfloor_cluster_picking_repack_mobile/static/src/js/i18n/fr.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why 2 separated files for such a small amount of translations?
With this approach we have:
* 3 requests to load the files
* 2 additional requests to load the translations

In such cases, I recommend to add the translations in the component file (which btw contextualizes them). Or in a single translation file (w/o using `load` anyway.